rpc/send.go

```diff
@@ -128,6 +129,12 @@ func Send(opts Options, method string, addrs []net.Addr, getArgs func(addr net.A
 			clients = append(clients, unhealthy[idx])
 		}
 	}
+
+	return doSend(clients, opts, method, addrs, getArgs, getReply)
+}
+
+func doSend(clients []*Client, opts Options, method string, addrs []net.Addr, getArgs func(addr net.Addr) interface{},
```

Comment:
In the naming conventions we've been using, a method like this should be named "sendInternal" instead of "doSend".